apps/examples/testcase/le_tc/network/Kconfig

```diff
@@ -49,6 +50,10 @@ config TC_NET_SOCKET
 	bool "socket() api"
 	default n
 
+config TC_NET_PKT
```

Comment:
Description is pbuf but config name is pkt. What's the correct?

Reply:
corrected